fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerTest.java

```diff
@@ -2620,4 +2636,25 @@ public void markReadable() {
             assertTrue(indexer.asyncToSync(FDBStoreTimer.Waits.WAIT_ONLINE_BUILD_INDEX, indexer.markReadableIfBuilt()));
         }
     }
+
+    @Test
+    public void testConfigLoader() {
+        Index index = runAsyncSetup();
+        try (OnlineIndexer indexBuilder = OnlineIndexer.newBuilder()
+                .setDatabase(fdb).setMetaData(metaData).setIndex(index).setSubspace(subspace)
+                .setConfigLoader(old ->
+                        CompletableFuture.completedFuture(OnlineIndexer.Config.newBuilder()
+                                .setLimit(100)
+                                .setMaxRetries(3)
+                                .setRecordsPerSecond(10000)
+                                .build()))
+                .setMdcContext(ImmutableMap.of("mdcKey", "my cool mdc value"))
```

Comment:
Probably don't want this line unless we're using the mdc to avoid extra cruft in the test

Reply:
dropped.